From 0dac33e8532557ce10975f2b040725e0cd82feb4 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 11 Jun 2016 22:12:51 +0000
Subject: [PATCH 125/131] printk end of boot

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index d35efcc10474..157185d47824 100644
--- a/init/main.c
+++ b/init/main.c
@@ -959,6 +959,7 @@ static int __ref kernel_init(void *unused)
 		       ramdisk_execute_command, ret);
 	}
 
+	printk(KERN_ERR "Executing init\n");
 	/*
 	 * We try each of these until one succeeds.
 	 *
-- 
2.14.2

